.github/workflows/backwards_compatibility_tests_workflow.yml

```diff
@@ -46,8 +46,8 @@ jobs:
     strategy:
       matrix:
         java: [ 11, 17 ]
-        bwc_version: [ "1.3.2", "2.0.0", "2.1.0", "2.2.0" ]
-        opensearch_version: [ "2.3.0-SNAPSHOT" ]
+        bwc_version: [ "2.4.0-SNAPSHOT" ]
```

Comment:
why "2.0.0", "2.1.0", "2.2.0" got removed?

Reply:
For rolling upgrade tests we need to upgrade from most recent version of previous major (2.4.0) to next major version (3.0.0)